ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala

```diff
@@ -64,33 +115,41 @@ class CommandStaticTimeIT
 
   }
 
-  private def submissionWithTime(ctx: LedgerContext, time: Instant): SubmitRequest = {
+  private def submissionWithTime(time: Instant): SubmitRequest = {
     val let = fromInstant(time)
     val mrt = let.update(_.seconds.modify(_ + ttlSeconds))
     submitRequest
       .update(
         _.commands.ledgerEffectiveTime := let,
         _.commands.maximumRecordTime := mrt,
-        _.commands.ledgerId := ctx.ledgerId.unwrap,
+        _.commands.ledgerId := unwrappedLedgerId,
         _.commands.commandId := newCommandId()
       )
   }
 
   // format: off
```

Comment:
Is there a reason we're not letting the formatter do its thing?

Reply:
I can disable this and see if it works, all I've done was adding a missing `// format: off`. :wink:

Reply:
My best guess is a `scalafmt` search state explosion, but probably this has been mitigated by your recent dependency bump and it's worth trying. :slightly_smiling_face:

Reply:
> Search state exploded around line 135
Confirmed, I'll have to leave this as is, but at least the rest of the file will be formatted automatically. :slightly_smiling_face:

Reply:
Fair. 😄